edsdme/blocks/announcements-preview/announcements-preview.js

```diff
@@ -0,0 +1,147 @@
+import { getCaasUrl } from '../../scripts/utils.js';
+import { getConfig } from '../utils/utils.js';
+
+export default async function init(el) {
+  const config = getConfig();
+  const newestCards = [];
+
+  const block = {
+    el,
+    collectionTag: '"caas:adobe-partners/collections/announcements"',
+    ietf: config.locale.ietf,
+  };
+
+  const blockData = {
+    caasUrl: getCaasUrl(block),
+    ietf: config.locale.ietf,
+    title: '',
+    buttonText: '',
+    link: '',
+    empty: 'Currently, there are no partner announcements',
+  };
+
+  const app = document.createElement('announcements-preview');
```

Comment:
Maybe let's not use specific element for this block, just use div like the rest of MILO blocks, because we are not using WebComponents here.